src/Tribe/PUE/Utility.php

```diff
@@ -104,6 +107,19 @@ public function to_wp_format() {
 				$update->upgrade_notice = $this->upgrade_notice;
 			}
 
+			// Support custom $update properties coming straight from PUE
+			if ( ! empty( $this->custom_update ) ) {
+				$custom_update = get_object_vars( (object) $this->custom_update );
```

Comment:
Why cast as Object than use `get_object_vars()`?

Reply:
It doesn't have to be there, I can remove the cast. It comes through as an object (or nothing at all).

Reply:
Removed the cast